### PR TITLE
Models

### PR DESCRIPTION
**Pull request by Crash1212**

### CODE REVIEW

File: src/components/model-form/model-form.js [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+              />
+              {touched[`${lang}Name`] && errors[`${lang}Name`] && (
+                <div className={styles.inputError}>{errors[`${lang}Name`]}</div>
+              )}
+              <TextField
+                data-cy={`${lang}Description`}
+                id={`${lang}Description`}
+                className={styles.textField}
+                variant='outlined'
+                label='Опис'
```
Comment (TorskyM) [inline src/components/model-form/model-form.js:231]: Move input labels to the config file

File: src/components/model-form/model-form.js [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+            className={styles.tabs}
+            value={tabsValue}
+            onChange={handleTabsChange}
+            aria-label='simple tabs example'
+          >
+            {languageTabs}
+          </Tabs>
+        </AppBar>
+        {languages.map((lang, index) => (
+          <TabPanel key={index} value={tabsValue} index={index}>
```
Comment (TorskyM) [inline src/components/model-form/model-form.js:210]: Do not use `index` as a key value

File: src/components/model-form/model-form.js [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+          className={styles.returnButton}
+          data-cy='goBackButton'
+        >
+          {config.buttonTitles.GO_BACK_TITLE}
+        </Button>
+        <SaveButton
+          className={styles.saveButton}
+          data-cy='save'
+          type='submit'
+          title='Зберегти'
```
Comment (TorskyM) [inline src/components/model-form/model-form.js:265]: Import `Зберегти` title from config file

File: src/configs/index.js [hunk 4]
```diff
@@ -287,10 +313,19 @@ export const config = {
...
+      category: 'Категорія',
+      available: 'Доступний',
+      show: 'Показати',
+      avatarText: 'Фото',
+      priority: 'Пріорітет'
     }
   },
   IMG_URL: 'https://horondi.blob.core.windows.net/horondi/images/',
   patternImageLink: `https://horondi.blob.core.windows.net/horondi/images/`,
+  imageLink: `https://horondi.blob.core.windows.net/horondi/images/`,
```
Comment (TorskyM) [inline src/configs/index.js:328]: This value is already assigned on line `326`, do we really need to duplicate it?

File: src/pages/model/model-page/model-page.js [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+      MODEL_REMOVE_TITLE,
+      MODEL_REMOVE_MESSAGE,
+      MODEL_REMOVE_TITLE
+    );
+  };
+
+  const changeHandler = (e, value) => dispatch(setModelsCurrentPage(value));
+
+  const modelItems =
+    list !== undefined
```
Comment (TorskyM) [inline src/pages/model/model-page/model-page.js:70]: You could simplify this check to `list ?`

File: src/pages/model/model-page/model-page.js [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+      ? list.map((modelItem, index) => (
+        <TableContainerRow
+          image={
+            modelItem.images
+              ? `${config.imageLink}${modelItem.images.thumbnail}`
+              : ''
+          }
+          key={index}
```
Comment (TorskyM) [inline src/pages/model/model-page/model-page.js:78]: Don't use the index as a key value

File: src/pages/model/model-page/model-page.js [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+          id={modelItem._id}
+          name={modelItem.name[0].value}
+          category={modelItem.category.name[0].value}
+          show={modelItem.show ? 'Так' : 'Ні'}
```
Comment (TorskyM) [inline src/pages/model/model-page/model-page.js:82]: Move these values to the config file

File: src/redux/model/model.operations.js [hunk 8]
```diff
@@ -0,0 +1,271 @@
...
+    context: { headers: { token } },
+    mutation: gql`
+      mutation($id: ID!) {
+        deleteModel(id: $id) {
+          ... on Model {
+            _id
+            name {
+              lang
+              value
+            }
```
Comment (TorskyM) [inline src/redux/model/model.operations.js:117]: Do we need to return all this model data in the `delete` mutation?

**Review by TorskyM — APPROVED** [review]

**Review by ecl1pseo — APPROVED** [review]

**Review by INR4GE — APPROVED** [review]

**Pull request merged**